From 2abded877782d9a8bac96e6f1f3a53ca3396f238 Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@gmail.com>
Date: Wed, 2 May 2018 09:08:21 -0700
Subject: [PATCH] TRITON-356 VMAPI should use node v6.x Reviewed by: Marsell
 Kukuljevic <marsell@joyent.com> Approved by: Marsell Kukuljevic
 <marsell@joyent.com>

---
 Makefile     | 4 ++--
 package.json | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index e8bbe11..e617edb 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2017, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -24,7 +24,7 @@
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v4.6.1
+NODE_PREBUILT_VERSION=v6.14.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
 	# Allow building on other than sdc-minimal-multiarch-lts@15.4.1
diff --git a/package.json b/package.json
index fad45ac..9042f19 100644
--- a/package.json
+++ b/package.json
@@ -1,14 +1,14 @@
 {
   "name": "vmapi",
   "description": "VMs API",
-  "version": "9.5.1",
+  "version": "9.5.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
     "assert-plus": "1.0.0",
     "async": "0.7.0",
     "backoff": "2.5.0",
-    "bunyan": "1.8.1",
+    "bunyan": "1.8.12",
     "changefeed": "1.3.0",
     "cueball": "2.2.7",
     "dashdash": "1.14.1",
-- 
2.21.0

